core/actions/googleDriveImport/run.ts

```diff
@@ -158,6 +157,15 @@ export const run = defineRun<typeof action>(
 					relations,
 				});
 			}
+			await updatePub({
```

Comment:
This is my first time using `updatePub` in an action. Am I missing anything obvious with how I setup the call?

Reply:
nope, looks good to me. that looks like the right value format and passing on the `lastModifiedBy` like you did is correct